x/concentrated-liquidity/genesis_test.go

```diff
@@ -20,8 +20,10 @@ import (
 var (
 	testGenesisPools = []model.Pool{}
 	testGenesis      = types.GenesisState{
-		Params: types.Params{AuthorizedTickSpacing: []uint64{1, 10, 50}},
-		Pools:  []*codectypes.Any{},
+		Params: types.Params{
+			AuthorizedTickSpacing: []uint64{1, 10, 50},
+			AuthorizedSwapFees:    []sdk.Dec{sdk.MustNewDecFromStr("0.0001"), sdk.MustNewDecFromStr("0.0003"), sdk.MustNewDecFromStr("0.0005")}},
```

Comment:
Same comment about genesis - why do we export it if the expectation is to have them be as constants?

Reply:
@p0mvn do you think I am good to make a v16 upgrade folder so I can move the constants there?

Reply:
Ended up directly adding it to the default setting as mint does it https://github.com/osmosis-labs/osmosis/pull/4449/commits/02c67d25065d953315a3b4de4f88e99b41742865

Reply:
> do you think I am good to make a v16 upgrade folder so I can move the constants there?

Let's have this in the v15 upgrade handler please until we update e2e to perform an upgrade from v15 to v16.

I will port all the v16 logic from v15 handler as part of the post-release tasks.